### ISSUE TITLE
Polymer expression parsing: colons are allowed in property names

### ISSUE THREAD

**Issue by rictic** [Polymer/polymer-analyzer#592]
Polymer will accept syntax like: 

```html
<dt>quota</dt> <dd>[[auth.attributes.custom:quota]]</dd>
```

We shouldn't warn for this syntax.

**Comment by bennypowers** [comment]
Should add tokens regex in https://github.com/Polymer/polymer-analyzer/blob/5d865418783e56c7724411d2f37c2f974c492aea/src/javascript/javascript-parser.ts#L30

https://github.com/eslint/espree/blob/master/docs/tokens.md

**Comment by web-padawan** [comment]
BTW, when using data binding with colons, `polymer-build` fails with a really confusing output:

```
Error: 2 error(s) occurred during build.
  at BuildAnalyzer._done (/etc/bi-frontend/node_modules/polymer-build/lib/analyzer.js:229:36)
  at BuildAnalyzer.<anonymous> (/etc/bi-frontend/node_modules/polymer-build/lib/analyzer.js:189:26)
  at next (native)
  at fulfilled (/etc/bi-frontend/node_modules/polymer-build/lib/analyzer.js:17:58)
  at process._tickDomainCallback (internal/process/next_tick.js:135:7)
```
